dictionary/lib/dictionary.ex

```diff
@@ -1,5 +1,11 @@
 defmodule Dictionary do
-
-  defdelegate random_word(), to: Dictionary.WordList
-
+  use Application
```

Comment:
No deduction, but I personally prefer to have the application start up in a separate module (Dictionary.Application) that contains an inline supervisor start-up.